capturable/src/test/java/dev/shreyaspatil/capturable/controller/CaptureControllerTest.kt

```diff
@@ -55,6 +56,16 @@ class CaptureControllerTest {
         assertEquals(captureRequest.imageBitmapDeferred.isActive, true)
     }
 
+    @Test
+    fun captureRequests_collectedMultipleTimes() = runTest {
+        // Eagerly collect flow and cancel it to make sure we're subscribed to at least once.
+        asyncOnUnconfinedDispatcher { getRecentCaptureRequest() }.cancelAndJoin()
+
+        // When: Flow is collected again
+        asyncOnUnconfinedDispatcher { getRecentCaptureRequest() }.cancelAndJoin()
+        // Then: It should not crash
```

Comment:
Thanks for the change. Just a small suggestions to also assert whether values are getting emitted in the latest subscription or not:

```suggestion
        // Given: Eagerly collect flow and cancel it to make sure we're subscribed to at least once.
        val oldCaptureRequest = asyncOnUnconfinedDispatcher { getRecentCaptureRequest() }.cancel()

        // When: Flow is collected again and capture request is emitted
        val latestCaptureRequestDeferred = asyncOnUnconfinedDispatcher { getRecentCaptureRequest() }
        controller.captureAsync()

        // Then: Capture request should be get emitted with graphics layer
        assertEquals(latestCaptureRequestDeferred.await().imageBitmapDeferred.isActive, true)
```

Reply:
Do you want to assign `oldCaptureRequest` to the `cancel()` operation?

Reply:
Good catch, I think then it'll be fine if we do not collect it in a variable.

Reply:
Updated my suggestion, anyway it was unused 👍🏻

Reply:
I changed it in a separate commit and co-authored you: [e9981d7fd5e40c8653871cfd8e78ffd07a35a8cc](https://github.com/PatilShreyas/Capturable/pull/232/commits/e9981d7fd5e40c8653871cfd8e78ffd07a35a8cc). Otherwise the linter would fail due to no longer used `cancelAndJoin()`.

Reply:
Thank you ✅